phpBB/phpbb/db/migrator.php

```diff
@@ -215,13 +215,13 @@ protected function get_valid_name($name)
 			$prepended_name = ($name[0] == '\\' ? '' : '\\') . $name;
 			$prefixless_name = $name[0] == '\\' ? substr($name, 1) : $name;
 
-			if (isset($this->migration_state[$prepended_name]))
+			if (!isset($this->migration_state[$prepended_name]) && isset($this->migration_state[$prefixless_name]))
```

Comment:
Before the first check was not exist as in
https://github.com/phpbb/phpbb/pull/4492#issuecomment-269352344
but I want @marc1706  to be sure there are any side effects with this correction